internal/boxcli/secrets.go

```diff
@@ -0,0 +1,211 @@
+	)
+
+	return cmd
+}
+
+func secretsSetCmd(flags *secretsFlags) *cobra.Command {
+	return &cobra.Command{
+		Use:   "set <NAME1>=<value1> [<NAME2>=<value2>]...",
+		Short: "Securely store one or more environment variables",
+		Long:  "Securely store one or more environment variables. To test contents of a file as a secret use set=@<file>",
+		Args:  cobra.MinimumNArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return envsec.ValidateSetArgs(args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			box, err := devbox.Open(&devopt.Opts{
+				Dir:         flags.config.path,
+				Environment: flags.config.environment,
+				Stderr:      cmd.ErrOrStderr(),
+			})
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			secrets, err := box.Secrets(ctx)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			envID, err := secrets.EnvID()
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			return secrets.SetFromArgs(ctx, envID, args)
+		},
+	}
+}
+
+func secretsRemoveCmd(flags *secretsFlags) *cobra.Command {
+	return &cobra.Command{
+		Use:     "remove <NAME1> [<NAME2>]...",
+		Short:   "Remove one or more environment variables",
+		Aliases: []string{"rm"},
+		Args:    cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			box, err := devbox.Open(&devopt.Opts{
+				Dir:         flags.config.path,
+				Environment: flags.config.environment,
+				Stderr:      cmd.ErrOrStderr(),
+			})
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			secrets, err := box.Secrets(ctx)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			envID, err := secrets.EnvID()
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			return secrets.DeleteAll(ctx, envID, args...)
+		},
+	}
+}
+
+func secretsListCmd(commonFlags *secretsFlags) *cobra.Command {
+	flags := secretsListFlags{}
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls"},
+		Short:   "List all secrets",
+		Args:    cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			box, err := devbox.Open(&devopt.Opts{
+				Dir:         commonFlags.config.path,
+				Environment: commonFlags.config.environment,
+				Stderr:      cmd.ErrOrStderr(),
+			})
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			secrets, err := box.Secrets(ctx)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			envID, err := secrets.EnvID()
+			if err != nil {
+				return errors.WithStack(err)
+			}
+
+			vars, err := secrets.List(ctx, envID)
+			if err != nil {
+				return err
+			}
+
+			return envsec.PrintEnvVars(
+				vars, cmd.OutOrStdout(), flags.show, flags.format)
+		},
+	}
+
+	cmd.Flags().BoolVarP(
+		&flags.show,
+		"show",
+		"s",
+		false,
+		"Display the value of each environment variable (secrets included)",
```

Comment:
It feels odd to mention `environment variable` when the command is `devbox secrets`.

Can this be: `Display the value of the secret [default = false]`?